### PR TITLE
cloud-instance-workload-analysis-dashboard

### PR DESCRIPTION
**Pull request by ericanagle1**

### CODE REVIEW

File: src/boards/cloud-instance-workload-analysis/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+Publisher: JupiterOne
+---
+
+# CLOUD INSTANCE & WORLOAD ANALYSIS
+
+![sample-screenshot](board.png)
+
+Enter hostname, instanceId or ipAddress to visualize all the resources related to a compromised host/workload/instance. Review the privileged access to the instance and view the blast radius — in the example screenshot, see that two EC2 instances are in the same subnet, share the same SSH key, and are also an on-prem firewall (RTP-FW-01) allowed access to the instance.
```
Comment (erkangz) [inline src/boards/cloud-instance-workload-analysis/readme.md:9]: We should update this description a bit. 

The board answers 5 key questions for any given virtual instance or workload across all 3 major CSPs in seconds:

- What resources are connected to it?
- What else are in the same blast radius?
- How is it connected to the Internet (external attack path)?
- Who has admin / privileged access to it?
- What data stores does it have access to?

**Review by erkangz — CHANGES_REQUESTED** [review]

**Review by ericanagle1 — COMMENTED** [review]
Adjustment made on description

File: src/boards/cloud-instance-workload-analysis/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+Publisher: JupiterOne
+---
+
+# CLOUD INSTANCE & WORLOAD ANALYSIS
+
+![sample-screenshot](board.png)
+
+This board answers 5 key questions for any given virtual instance or workload across all 3 major CSPs in seconds:
```
Comment (erkangz) [inline src/boards/cloud-instance-workload-analysis/readme.md:9]:
```suggestion
This is an interactive dashboard that will prompt you to enter the hostname or IP address or instance ID of a cloud workload / instance.

This board answers 5 key questions for any given virtual instance or workload across all 3 major CSPs in seconds:
```

File: src/boards/cloud-instance-workload-analysis/readme.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+What resources are connected to it?
+What else are in the same blast radius?
+How is it connected to the Internet (external attack path)?
+Who has admin / privileged access to it?
+What data stores does it have access to?
```
Comment (erkangz) [inline src/boards/cloud-instance-workload-analysis/readme.md:15]:
```suggestion
- Is the workload / instance still active or online? What are its configurations?
- What resources are connected to it?
- What else are in the same blast radius?
- How is it connected to the Internet (external attack path)?
- Who has admin / privileged access to it?
- What data stores does it have access to?
```

**Review by erkangz — APPROVED** [review]

**Review by BrendanQuinnJ1 — APPROVED** [review]
Approved

**Pull request merged**